bin/fixtures/functions.php

```diff
@@ -11,94 +11,25 @@ function samltheme_parent_theme_enqueue_styles() {
-        /**
-         * Whether or not to automatically provision new WordPress users.
-         *
-         * When WordPress is presented with a SAML user without a
-         * corresponding WordPress account, it can either create a new user
-         * or display an error that the user needs to contact the site
-         * administrator.
-         *
-         * @param bool
-         */
-        'auto_provision'         => true,
-        /**
-         * Whether or not to permit logging in with username and password.
-         *
-         * If this feature is disabled, all authentication requests will be
-         * channeled through SimpleSAMLphp.
-         *
-         * @param bool
-         */
-        'permit_wp_login'        => false,
-        /**
-         * Attribute by which to get a WordPress user for a SAML user.
-         *
-         * @param string Supported options are 'email' and 'login'.
-         */
-        'get_user_by'            => 'email',
-        /**
-         * SAML attribute which includes the user_login value for a user.
-         *
-         * @param string
-         */
-        'user_login_attribute'   => 'uid',
-        /**
-         * SAML attribute which includes the user_email value for a user.
-         *
-         * @param string
-         */
-        'user_email_attribute'   => 'mail',
-        /**
-         * SAML attribute which includes the display_name value for a user.
-         *
-         * @param string
-         */
-        'display_name_attribute' => 'display_name',
-        /**
-         * SAML attribute which includes the first_name value for a user.
-         *
-         * @param string
-         */
-        'first_name_attribute' => 'first_name',
-        /**
-         * SAML attribute which includes the last_name value for a user.
-         *
-         * @param string
-         */
-        'last_name_attribute' => 'last_name',
-        /**
-         * Default WordPress role to grant when provisioning new users.
-         *
-         * @param string
-         */
-        'default_role'           => get_option( 'default_role' ),
-    );
-    $value = isset( $defaults[ $option_name ] ) ? $defaults[ $option_name ] : $value;
+add_filter( 'wp_saml_auth_option', function( $value, $option_name ){
+    if ( 'connection_type' === $option_name ) {
+        return 'internal';
+    }
+    if ( 'internal_config' === $option_name ) {
+        $value['idp']['entityId'] = home_url( '/simplesaml/saml2/idp/metadata.php' );
+        $value['idp']['singleSignOnService']['url'] = home_url( '/simplesaml/saml2/idp/SSOService.php' );
+        $value['idp']['x509cert'] = file_get_contents( ABSPATH . '/private/simplesamlphp/cert/saml.crt' );
+        $value['idp']['singleLogoutService']['url'] = home_url( '/simplesaml/saml2/idp/SingleLogoutService.php' );
+        return $value;
+    }
+    if ( 'user_login_attribute' === $option_name ) {
+        return 'urn:oid:0.9.2342.19200300.100.1.1';
```

Comment:
I sniffed it out of the SAML response. It looks like this is SAML 2 spec though: https://commons.lbl.gov/display/IDMgmt/Attribute+Definitions#AttributeDefinitions-uiduid